### PR TITLE
Fix #1113: Index files outside project directory

### LINKED ISSUE #1113: Index files outside project directory
Hi!

I was wondering if it is possible to index files outside project directory in order to have completion and code navigation. I'm working on projects that depend on other php files that are in a different location (these projects are not using composer).

Everyone here is using phpstorm and it is possible to add these files as 'External librairies' to have autocompletion of classes and methods and code navigation.

Thanks! :)

### PR DESCRIPTION
**Pull request by camilledejoye**
Fix #1113 

As discussed it's a simple update of the doc to be clearer on what the option is doing and how it works.

### CODE REVIEW

**Comment by elythyr** [comment]
Since you fixed the CI I forced push to trigger it and I think we are good to go.

File: doc/reference/indexer.rst [hunk 1]
```diff
@@ -149,13 +149,13 @@ Configuration
...
-- ``indexer.include_patterns``: List of glob patterns to include
-- ``indexer.exclude_patterns``: List of glob patterns to exclude
-- ``indexer.poll_time``: Poll time used for polling watchers (e.g. ``find``,
-  ``php``
-- ``indexer.buffer_time``: Time to wait to collect batch messages from
+- :ref:`param_indexer.include_patterns`: List of glob patterns to include
+- :ref:`param_indexer.exclude_patterns`: List of glob patterns to exclude
+- :ref:`param_indexer.stub_paths`: List of external paths to index
+- :ref:`param_indexer.poll_time`: Poll time used for polling watchers (e.g. ``find``, ``php``)
+- :ref:`param_indexer.buffer_time`: Time to wait to collect batch messages from
```
Comment (dantleech) [inline doc/reference/indexer.rst:158]: Are you aware that this documentation is generated?

Comment (elythyr) [reply doc/reference/indexer.rst:158]: No, it's generated from which other part of the documentation ?

Comment (dantleech) [reply doc/reference/indexer.rst:158]: https://github.com/phpactor/debug-extension/blob/master/lib/Model/ExtensionDocumentor.php

e.g. https://github.com/phpactor/indexer-extension/blob/master/lib/Extension/IndexerExtension.php#L95-L104

Comment (dantleech) [reply doc/reference/indexer.rst:158]: The command to generate:

```
phpactor development:configuration-reference
```

Comment (camilledejoye) [reply doc/reference/indexer.rst]: OK I don't know how I did it but somehow forced push my changed ha closed the PR and marked it as merged ><

I updated the `doc/reference/configuration.rst` file using the command, the PR for the indexer extension: https://github.com/phpactor/indexer-extension/pull/25

Comment (dantleech) [reply doc/reference/indexer.rst:158]: Cool - in general I would generate this file before release anyway, so no need to update it specifically.

**Comment by dantleech** [comment]
:+1:

**Pull request merged**